bin/ember-template-lint.js

```diff
@@ -340,18 +347,16 @@ async function run() {
     }
 
     if (options.updateTodo) {
-      let todoConfig = getTodoConfig(
-        options.workingDirectory,
-        getTodoConfigFromCommandLineOptions(options)
+      let [added, removed] = await linter.updateTodo(
+        linterOptions,
+        fileResults,
+        todoInfo.todoConfig
       );
 
-      let [added, removed] = await linter.updateTodo(linterOptions, fileResults, todoConfig);
+      todosToAdd = true;
```

Comment:
This doesn't seem needed, can't it just look at todoInfo.added down below?

Reply:
Also, why is it `true`? Seems like it's possible to call `linter.updateTodo` and have that return `[0,0]` (nothing added or removed).

Unless I'm missing something, it seems that using the numbers directly is better.

Reply:
We don't want to look at added because 0 is a valid value as we allow you to output `0 todos created...`. Using the flag is a cleaner way to determine if we should have todo output at all.

Reply:
If that is the case, then why do we need the boolean at all?

Reply:
We don't...

:)